x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/search/SearchApplication.java

```diff
@@ -52,8 +52,10 @@ public class SearchApplication implements Writeable, ToXContentObject {
     public static final String NO_TEMPLATE_STORED_WARNING = "Using default search application template which is subject to change. "
         + "We recommend storing a template to avoid breaking changes.";
 
+    public static final String NO_INDICES_WARNING = "Indices are missing. "
```

Comment:
Since we're moving away from storing indices associated with search applications, the system of record should be the fact that the alias exists. If the alias exists, we by definition have to have at least one index. I realize this is a bit of a change to the way we think about it but I think it's more clear to communicate what's really happening under the hood here.

Reply:
@kderusso Can we rewrite the warning as follows?

`Alias is missing for the search application`

Reply:
That would work - also, renaming to NO_ALIAS_WARNING 👍

Reply:
I like both suggestions, thank you!